code/Data exploration.R

```diff
@@ -199,20 +254,22 @@ data$SUMMER <- if_else(data$SEASON == 'SUMMER',1,0)
 data$AUTUMN <- if_else(data$SEASON == 'AUTUMN',1,0)
 data$WINTER <- if_else(data$SEASON == 'WINTER',1,0)
```

Comment:
didn't we decide to only include flags for summer and winter? (default being not summer or winter)

Reply:
That is correct, I've made scatterplot to fortify the reasoning too - I just kept that fields in for now. Will create a new table called data_final to be used for modelling, that utilizes only fields we're interested in.